src/GitVersionCore.Tests/VersionCalculation/Strategies/MergeMessageBaseVersionStrategyTests.cs

```diff
@@ -40,7 +40,6 @@ public void ShouldNotAllowIncrementOfVersion()
         [TestCase("Merge branch 'Releases-0.2.0'", true, "0.2.0")]
         [TestCase("Merge branch 'Releases/0.2.0'", true, "0.2.0")]
         [TestCase("Merge branch 'release-4.6.6' into support-4.6", true, "4.6.6")]
-        [TestCase("Merge branch 'release-10.10.50'", true, "10.10.50")]
```

Comment:
Why is this test case deleted?

Reply:
I deleted the wrong one I think. Line 37 and line 40 are the same test case, it was supposed to be one of those! Thanks for spotting that one.

Reply:
You know what, I just checked it again.

```
line35: [TestCase("Merge branch 'release-10.10.50'", true, "10.10.50")]
line43: [TestCase("Merge branch 'release-10.10.50'", true, "10.10.50")]
```
I think I hit the correct one the first time. line 37 and 40 have an "s" different, but 35 and 43 are identical. I removed line 43 in the commit there. Sorry about the confusion.

Reply:
Indeed. Good catch.